pytket/extensions/qiskit/backends/aer.py

```diff
@@ -660,6 +663,71 @@ def __init__(self, n_qubits: int = 40) -> None:
         ]
 
 
+class AerDensityMatrixBackend(_AerBaseBackend):
+    """
+    Backend for running simulations on the Qiskit Aer density matrix simulator.
+
+    :param noise_model: Noise model to apply during simulation. Defaults to None.
```

Comment:
Should these be defined here or in the init? I'm not sure what our standard is

Reply:
Seems to me that we document this in the class docstring rather than the `__init__` method. I don't have a strong prefernce either way.